scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -48,6 +48,12 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
                                 </div>
                             </div>
                             <div class="field">
+                                <label translate>Desk language</label>
+                                <input type="text" class="line-input fullwidth-input"
```

Comment:
ok, so, maybe I can add list of languages that we have set in dictionaries? I was thinking about this, but on some planning we had discussion about this and conclusion was to left input box...

What you think about setting language list from dictionaries?